build-system/tasks/e2e/index.js

```diff
@@ -163,4 +173,5 @@ e2e.flags = {
     'Options are `puppeteer` or `selenium`. Default: `selenium`',
   'headless': '  Runs the browser in headless mode',
   'debug': '  Prints debugging information while running tests',
+  'report': '  Write test result report to a local file',
```

Comment:
Didn't we use `report-results` or something elsewhere, or was this the same flag?

Reply:
The flag for the tasks is report; the uploading gulp task is is test-report-upload -- see https://github.com/ampproject/amphtml/pull/29495#discussion_r460230722